pkg/cliplugins/workspace/plugin/kubeconfig.go

```diff
@@ -300,13 +301,32 @@ func (kc *KubeConfig) CreateWorkspace(ctx context.Context, workspaceName string,
 		return fmt.Errorf("current URL %q does not point to cluster workspace", config.Host)
 	}
 
+	var structuredWorkspaceType tenancyv1alpha1.ClusterWorkspaceTypeReference
+	if workspaceType != "" {
+		parts := strings.Split(workspaceType, "|")
```

Comment:
this is not a notation for users.

We talked about this elsewhere to use `root:ws:type`, i.e. like filenames and paths.

Reply:
That makes it not parsable trivially though, I'm against that ...